tests/ui/abi/compatibility.rs

```diff
@@ -1,16 +1,174 @@
 // check-pass
+// revisions: host
+// revisions: arm
+//[arm] compile-flags: --target arm-unknown-linux-gnueabi
+//[arm] needs-llvm-components: arm
+// revisions: aarch64
+//[aarch64] compile-flags: --target aarch64-unknown-linux-gnu
+//[aarch64] needs-llvm-components: aarch64
+// revisions: s390x
+//[s390x] compile-flags: --target s390x-unknown-linux-gnu
+//[s390x] needs-llvm-components: s390x
```

Comment:
```suggestion
//[s390x] needs-llvm-components: systemz
```

Reply:
I took this from https://github.com/rust-lang/rust/pull/116004. So if this is wrong then that PR also gets it wrong.